restaurant/forms.py

```diff
@@ -4,29 +4,22 @@
 
 class QuestionnaireForm(forms.Form):
     restaurant_business_id = forms.CharField(label="restaurant_id")
+    user_id = forms.CharField(label="user_id")
     safety_level = forms.CharField(label="safety_level")
+    saved_on = forms.CharField(label="saved_on")
```

Comment:
as suggested, we could use DateTimeField instead of CharField. i.e. (auto_add_now=True)

Reply:
Fixed using timezone.now